core/src/main/java/org/elasticsearch/search/internal/ShardSearchLocalRequest.java

```diff
@@ -93,11 +95,12 @@ public ShardSearchLocalRequest(ShardId shardId, String[] types, long nowInMillis
         this.nowInMillis = nowInMillis;
         this.aliasFilter = aliasFilter;
         this.shardId = shardId;
+        //TODO default strict field here?
```

Comment:
...or add a `strict` parameter? (constructor is currently called from ValidateQuery and Explain actions).
Incidentally, where would be a good home to store a global constant for DEFAULT_STRICT_POLICY?